src/Type/WPConnectionType.php

```diff
@@ -140,6 +139,14 @@ public function __construct( array $config, TypeRegistry $type_registry ) {
 
 		$this->validate_config( $config );
 
+		/**
+		 * Bail if one of the connection types has been excluded from the schema.
+		 */
+		$excluded_types = $type_registry->get_excluded_types();
+		if ( in_array( ucfirst( $config['fromType'] ), $excluded_types, true ) || in_array( ucfirst( $config['toType'] ), $excluded_types, true ) ) {
```

Comment:
instead of `ucfirst()` here, again, I think we should do `strtolower()` as it normalizes the same way the type loader does in the type registry